ADAL/src/utils/ADHelpers.m

```diff
@@ -327,7 +328,9 @@ + (NSString*)canonicalizeAuthority:(NSString *)authority
     NSURL* url = [NSURL URLWithString:trimmedAuthority];
     if (!url)
     {
-        AD_LOG_WARN(nil, @"The authority is not a valid URL %@", authority);
+        AD_LOG_WARN(nil, @" The authority is not a valid URL authority: %@", [ADAuthorityUtils isKnownHost:[authority adUrl]] ? [authority adUrl].host : @"unknown host");
```

Comment:
nit: we're logging host and not authority

Reply:
fixed